myapp/config/routes.rb

```diff
@@ -11,5 +11,15 @@
   delete 'logout', to: 'sessions#destroy'
   # index page
   root 'tasks#index'
+  # admin page
```

Comment:
How about organizing like this?

```
namespace :admin do
  get '/', to: 'users#admin', as: :admin
  post '/', to: 'users#create_user'
  get ':id/tasks', to: 'users#user_tasks', as: :user_tasks
  get 'new', to: 'users#new_user'
end
```

Reply:
Changed it in dfb3467
The `scope` may be the one you refer?

Reply:
`scope` is fine! We can use both `scope`/`namespace`. Difference is directory structure.
https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing

Reply:
I used namespace at first but error occurred saying sth like `not initialized Admin constant` and then I changed to `scope`

Reply:
I see!
By using `namespace`, controller directory should be `controllers/admin/users_controller.rb`. That may be the reason why error occur.

Reply:
> place these controllers under the app/controllers/admin directory

Thank you now I know why that error happened!